src/omv/common/sensor.h

```diff
@@ -173,15 +173,11 @@ typedef enum {
     SENSOR_ERROR_JPEG_OVERFLOW          = -20,
 } sensor_error_t;
 
-#define SENSOR_HW_FLAGS_VSYNC           (0) // vertical sync polarity.
-#define SENSOR_HW_FLAGS_HSYNC           (1) // horizontal sync polarity.
-#define SENSOR_HW_FLAGS_PIXCK           (2) // pixel clock edge.
-#define SENSOR_HW_FLAGS_FSYNC           (3) // hardware frame sync.
-#define SENSOR_HW_FLAGS_JPEGE           (4) // hardware JPEG encoder.
-#define SENSOR_HW_FLAGS_RGB565_REV      (5) // byte reverse rgb565.
-#define SENSOR_HW_FLAGS_GET(s, x)       ((s)->hw_flags &  (1<<x))
-#define SENSOR_HW_FLAGS_SET(s, x, v)    ((s)->hw_flags |= (v<<x))
-#define SENSOR_HW_FLAGS_CLR(s, x)       ((s)->hw_flags &= ~(1<<x))
+// Bayer patterns.
+#define SENSOR_HW_FLAGS_BAYER_BGGR      (0) // Default Bayer pattern.
```

Comment:
It's not an enum but this is for the sensor bayer format set in sensor drivers, the image bayer format will have different constants defined in imlib.h.. It will make more sense with the next PR.

Reply:
I'm just saying other bayer formats may appear. So, it's not just 4 forever.

Reply:
@kwagyeman you can easily expand the bit-field if needed, we have like 18-bits left unused.